apps/rod/sections/summary-data-sections.js

```diff
@@ -14,6 +14,14 @@ module.exports = {
       step: '/sponsor-type',
       field: 'sponsor-type'
     },
+    {
+      step: '/about-application',
+      field: 'date-of-application'
```

Comment:
date format has be added for this date 
`parse: d => d && dateFormater.format(new Date(d))`